drivers/SRCommand.py

```diff
@@ -299,6 +299,14 @@ def _run(self, sr, target):
             finally:
                 os.unsetenv('THIN_STATE_FILE_ATTACH')
 
+        elif self.cmd == 'vdi_detach_from_config':
+            extras= {}
+            if target.sr.driver_config.get("ATTACH_FROM_CONFIG_WITH_TAPDISK"):
+                target = blktap2.VDI(self.vdi_uuid, target, self.driver_info)
+                extras['deactivate'] = True
+                extras['caching_params'] = caching_params
+            target.detach(self.params['sr_uuid'], self.vdi_uuid, **extras)
```

Comment:
From my code reading, the flag ``ATTACH_FROM_CONFIG_WITH_TAPDISK`` doesn't mean whether ``attach/detach_from_config`` is supported, rather it means whether tapdisk should be involved in the attach/detach process.

This is corresponding to the logic in the existing ``attach_from_config`` definition right above --- where when such flag is present, the target is re-instantiated by ``blktap2.VDI(...)`` before the attach, otherwise the attach method will still be called, just on plain VDI object instead.